### ISSUE TITLE
Don't overwrite .env.deploy.production

### ISSUE THREAD

**Issue by kpfefferle** [pagefront/ember-pagefront#14]
During the initial running of `ember install ember-pagefront --app=[appname]`, I get the same ember diff workflow that I see when updating ember-cli. This is good because I can choose to keep other custom settings I may have inside `.env.deploy.production`.

However, somewhere along the line (I believe it's when I run the `ember login` command to enable deploys to staging and development), my `.env.deploy.production` gets entirely overwritten with a version that only contains `PAGEFRONT_KEY`.

I would suggest that ember-pagefront not completely wipe out the user's local `.env.deploy.*` files but instead just append `PAGEFRONT_KEY` to them (or change that one env variable). Though the primary use case for PageFront would indeed have that key be the only value in the `.env.deploy.*` files, there are scenarios when some users may have other variables stored in this file as well.

**Comment by jasonkriss** [comment]
You are absolutely correct @kpfefferle. Thanks for pointing that out. Because the primary use case would only have the the Pagefront key, I never even considered the case where there would be something else there. We'll get this fixed. Thanks again!
